source/server/listener_manager_impl.cc

```diff
@@ -96,6 +118,30 @@ ListenerImpl::ListenerImpl(const envoy::api::v2::Listener& config, ListenerManag
   // filter chain #1308.
   ASSERT(config.filter_chains().size() >= 1);
 
+  if (config.listener_filters().size()) {
+    listener_filter_factories_ =
+        parent_.factory_.createListenerFilterFactoryList(config.listener_filters(), *this);
+  }
+  // Add original dst listener filter if 'use_original_dst' flag is set.
```

Comment:
Per my comment in the data-plane-api repo, can we deprecate the use_original_dst field in DEPRECATED.md and docs, and push people towards using the filter directly.

Reply:
Done.